### PR TITLE
ASSETS-34095 - Guidance to avoid UnrecognizedPropertyExceptions

### PR DESCRIPTION
**Pull request by ireasor**
## Description
- Adding guidance to set FAIL_ON_UNKNOWN_PROPERTIES to false in JSON parsing libraries.
- Adding a link to our end-to-end usage guide in Experience League.

## Motivation and Context
It will help customers to prevent runtime exceptions from occurring on API updates.  The link to Experience League will make it easier for customers to find the guidance when browsing the How To materials on developer.adobe.com.

## How Has This Been Tested?
Ran `yarn lint`, `yarn test:links`, and `yarn start`.  Browsed the modified pages locally to ensure that they rendered the content correctly.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: src/pages/guides/index.md
Comment (guillaumecarlino) [inline src/pages/guides/index.md:1]: I would change the text to something like this:
Have a look at [AEM Assets events for PIM integration](https://experienceleague.adobe.com/en/docs/experience-manager-learn/cloud-service/aem-eventing/examples/assets-pim-integration) to learn the steps that are needed to onboard the OpenAPI-based APIs and events for the use case of importing metadata from a third-party system whenever a new asset is uploaded into the DAM.

File: src/pages/guides/how-to/index.md
Comment (guillaumecarlino) [inline src/pages/guides/how-to/index.md:1]: There seem to be a couple of double spaces. Also maybe add quotes to DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES (like this: `DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES`) to better highlight it.

File: src/pages/guides/index.md
Comment (ireasor) [reply src/pages/guides/index.md:1]: Thanks, updated and added a sentence to clarify the value of this article for those with other use cases as well.

File: src/pages/guides/how-to/index.md
Comment (ireasor) [reply src/pages/guides/how-to/index.md:1]: The double-space after a period is something that I need to un-learn from school ;)

**Comment by ireasor** [comment]
@guillaumecarlino - any ideas about the failed validation?  It looks like an issue outside the context of this change, since it is complaining about the Node version.

**Review by cziegeler — APPROVED** [review]

**Comment by guillaumecarlino** [comment]
@ireasor the Node thing is only a warning. It was failing because of this error:
`Files should end with a single newline character`
I added one in the problematic file and it's back to green.

**Pull request merged**